klog.go

```diff
@@ -800,10 +800,12 @@ func kvListFormat(b *bytes.Buffer, keysAndValues ...interface{}) {
 			v = missingValue
 		}
 		b.WriteByte(' ')
-		if _, ok := v.(fmt.Stringer); ok {
+
+		switch v.(type) {
+		case string, ObjectRef:
 			b.WriteString(fmt.Sprintf("%s=%q", k, v))
-		} else {
-			b.WriteString(fmt.Sprintf("%s=%#v", k, v))
+		default:
+			b.WriteString(fmt.Sprintf("%s=%+v", k, v))
```

Comment:
```suggestion
			if _, ok := v.(fmt.Stringer); ok {	
			    b.WriteString(fmt.Sprintf("%s=%q", k, v))
			} else {
			    b.WriteString(fmt.Sprintf("%s=%+v", k, v))
			}
			
			
```
We cannot remove support for fmt.Stringer

Reply:
Yes, you are right, if add new struct with String() method in the future.